db/migrate/20170419121747_create_restaurants.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRestaurants < ActiveRecord::Migration[5.0]
+  def change
+    create_table :restaurants do |t|
+      t.string :name
```

Comment:
Please add some input validation. Name of restaurant should probably be unique, most of the other inputs also have limitations.